diff -urN gnome-screensaver-0.0.9.orig/configure.ac gnome-screensaver-0.0.9/configure.ac
--- gnome-screensaver-0.0.9.orig/configure.ac	2005-08-24 17:44:41.294062536 +0200
+++ gnome-screensaver-0.0.9/configure.ac	2005-08-24 17:44:55.933836952 +0200
@@ -1372,7 +1372,7 @@
 AC_SUBST(LDFLAGS)
 
 # Saver engine stuff
-SLIDESHOW_COSMOS_DIR="${EXPANDED_DATADIR}/pixmaps/backgrounds/cosmos"
+SLIDESHOW_COSMOS_DIR="${EXPANDED_DATADIR}/gnome-screensaver/themes/cosmos"
 
 AC_SUBST(SLIDESHOW_COSMOS_DIR)
 
