--- gnome-screensaver-3.6.1/configure.ac.systemd	2018-03-13 22:36:05.069234255 +0100
+++ gnome-screensaver-3.6.1/configure.ac	2018-03-13 22:37:02.587176254 +0100
@@ -602,7 +602,9 @@ AC_ARG_WITH(systemd,
 
 PKG_CHECK_MODULES(SYSTEMD,
                   [libsystemd-login libsystemd-daemon],
-                  [have_systemd=yes], [have_systemd=no])
+                  [have_systemd=yes], [PKG_CHECK_MODULES(SYSTEMD,
+                					 [libsystemd],
+                					 [have_systemd=yes], [have_systemd=no])])
 
 if test "x$with_systemd" = "xauto" ; then
         if test x$have_systemd = xno ; then
