diff -urN gnome-screensaver-0.0.13.orig/configure.ac gnome-screensaver-0.0.13/configure.ac
--- gnome-screensaver-0.0.13.orig/configure.ac	2005-09-12 17:28:46.000000000 +0200
+++ gnome-screensaver-0.0.13/configure.ac	2005-09-15 13:07:06.125298408 +0200
@@ -1015,16 +1015,11 @@
   xscreensaverdir="$with_xscreensaverdir"
 fi
 
-if test -d "$xscreensaverdir"; then
   AC_DEFINE_UNQUOTED(XSCREENSAVER_CONFIG_DIR, "$xscreensaverdir", [Define to the directory containing XScreensaver configuration files])
   AC_MSG_RESULT([$xscreensaverdir])
-else
-  AC_MSG_RESULT([no])
-fi
 
 # Path to xscreensaver hack executables
 AC_ARG_WITH(xscreensaverhackdir, AC_HELP_STRING([--with-xscreensaverhackdir=dir], [Look for xscreensaver hack executables in directory],))
-if test -d "$xscreensaverdir"; then
   AC_MSG_CHECKING([for location of xscreensaver hack executables])
   xscreensaverhackdir=
   if test "x$with_xscreensaverhackdir" = "xyes" -o "x$with_xscreensaverhackdir" = "x"; then
@@ -1042,16 +1037,8 @@
   elif test "x$with_xscreensaverhackdir" != "xno"; then
     xscreensaverhackdir="$with_xscreensaverhackdir"
   fi
-  if test -d "$xscreensaverhackdir"; then
     AC_DEFINE_UNQUOTED(XSCREENSAVER_HACK_DIR, "$xscreensaverhackdir", [Define to the directory containing XScreensaver hack executables])
     AC_MSG_RESULT([$xscreensaverhackdir])
-  else
-    AC_MSG_RESULT([no])
-    AC_MSG_WARN([using XScreensaver config files, but could not find directory containing hacks])
-  fi
-else
-  AC_MSG_WARN([ignoring --with-xscreensaverhackdir because directory for XScreensaver configuration files was not found])
-fi
 
 #       Check for PAM.
 
